extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -257,6 +262,42 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 
   /**
+   * If requestsFrameUrl (and optionally dataHash as well) are specified in
+   * config, check whether third-party iframe already exists, and if not,
+   * create it.
+   * @private
+   */
+  handleRequestsFrameUrl_() {
+    if (!this.config_['requestsFrameUrl']) {
+      return;
+    }
+    const requestsFrameKey = this.buildRequestsFrameUrlWithDataHash(
```

Comment:
We expect different hash values per slot so this will not work.  Instead, you should base this purely on the frame url or perhaps some sort of type name (e.g. 'moat')